contribution.md

```diff
@@ -43,30 +41,30 @@ If appropriate demo or sample are absent, you must provide your own demo (C++ or
 ```
     -i "<path>"              Optional. Path to a input file or directory (for multiple inferences). By default input must be generated randomly.
     -m "<path>"              Required. Path to an .xml file with a trained model
-    -d "<device>"            Optional. Target device for model inference. Usually CPU and GPU. By default CPU
-    -no_show                 Optional. Do not show inference result.
+    -d "<device>"            Optional. Target device for model inference. By default CPU
+    -no_show                 Optional. Do not launch GUI window to visualize inference results. Needed for demo CI tests automation.
 ```
 Also you can add all necessary parameters for inference your model.
 
 ## Accuracy validation
 
-To run accuracy validation, use [Accuracy Checker](./tools/accuracy_checker/README.md) tool, provided with repository. Doing this very simple if model task from supported. You must only create accuracy validation configuration file in this case. 
+To run accuracy validation, use [Accuracy Checker](./tools/accuracy_checker#testing-new-models) tool, provided with repository. It is simple if model task already supported by Accuracy Checker. You need only create Accuracy Checker configuration file, which contain necessary parameters to do accuracy validation (specify dataset and annotation, pre- and post processing parameters, accuracy metric to compute and so on).
```

Comment:
I meant a little different. When you speak that tool should be used, it is correct to use link on full README.
Where you discrabe which steps should be done, you need to linked testing-new-model section or just use steps description from it.

Reply:
Done

Reply:
as @eaidova says, the link to Accuracy Checker should be link to README do, then, you may add as a last pharse something like this: See more details by testing new models description [put here link to testing-new-model]

Reply:
Already done, please look at last commit